### ISSUE TITLE
Create Broker View

### ISSUE THREAD

**Issue by duncangraham** [hivemindworks/argo#8]
There should be a view that allows a broker to see the buses and make sure that they get booked.

I'm not sure how to best execute this-- what would you like from this interface @markstm17? a list of buses, the ability to check them off as they've been successfully booked, anything else?

**Comment by markstm17** [comment]
When you say Broker, you mean event planner right Dunc?

**Comment by markstm17** [comment]
Or do you mean me (Or eventually the bus company or bus driver)?

**Comment by duncangraham** [comment]
@markstm17 nope, I mean the person (you, right now) that has to make sure that the buses created in the app turn into real life buses

**Comment by markstm17** [comment]
@duncangraham 

Gotcha. 

Here's what I'll need:

-Date of Bus
-Pick Up Location of Bus
-Pick Up time of Bus
-Destination
-Total Seats/spots available (Maximum-for the sake of it right now, lets say 50 for everyone-will obviously change in future)
-Seats Reserved or Booked
-Seats still Open

-We can keep filling the seats up until the bus driver physically comes to get the group. If there are 45/50 spots booked and paid for at 1:30 and the bus driver comes to pick up at 2:00, and within that half hour 1-5 more people decide to come or haven't paid yet, we want to give them the ability to do so. 

Obviously that will be the "sub details" of the list of buses. But I'd say the Date of the event, name of the event (Brewers vs. Cardinals at Miller Park for example), Pick up time and location and 40/50 seats in the bottom or top right hand corner to show a quick glimpse of seat availability

**Comment by markstm17** [comment]
@duncangraham I see what you're saying too. 

There can be 2 Check marks:

First is when minimum is hit (whether it's 10 or 15 people-The 'THRESHOLD') This threshold could change based on each bus. That will depends on the price template we have in place and the actual cost for the bus (which I'll know and have beforehand)

When it is 50/50 people on the bus who have booked and paid, or when the bus' wheels start rolling at 2:05 because the departure was 2:00pm then we can check mark that bus/event as FULL or STARTED or BEGUN or EN ROUTE, etc...
